cura/PrinterOutput/NetworkedPrinterOutputDevice.py

```diff
@@ -133,12 +131,6 @@ def _update(self) -> None:
 
             self.setConnectionState(ConnectionState.Closed)
 
-            # We need to check if the manager needs to be re-created. If we don't, we get some issues when OSX goes to
-            # sleep.
-            if time_since_last_response > self._recreate_network_manager_time:
```

Comment:
This will break osx going to standby and then going out of sleep mode again.

Reply:
I know, see my comments.